docs/Documentation/Managing_Data/Transferring_Files/globus.md

```diff
@@ -1,4 +1,4 @@
---- layout: default 
+--- la*For large data transfers between NLR's high-performance computing (HPC)out: default 
```

Comment:
I believe this edit was intended to modify line 7, not this line.
Revert line 1:
```suggestion
layout: default
```